### PR TITLE
Increase max username

### PR DESCRIPTION
**Pull request by moritz89**
Why:

- Increase allowable username length
- Support making migrations

This change addresses the need by:

- Add migrations
- Add script to make migrations
- Increase max username limit
- Update dependencies
- Update docs
- Fix bug for blank INTERNAL_URL

### CODE REVIEW

**Comment by moritz89** [comment]
Combines currently open MRs into one

**Review by carlkibler — APPROVED** [review]

**Comment by moritz89** [comment]
@uw-rvitorino @diogosilva30 Is carlkibler part of your team? I still need an approval from someone with write access

**Comment by moritz89** [comment]
@uw-rvitorino @diogosilva30 do you have time to have a look?

**Review by uw-rvitorino — APPROVED** [review]

**Pull request merged**
